src/test/regress/expected/drop_partitioned_table.out

```diff
@@ -0,0 +1,469 @@
+ logicalrelid | shardid
+---------------------------------------------------------------------
+ parent       |  720000
+ parent       |  720001
+ parent       |  720002
+ parent       |  720003
+ child1       |  720004
+ child1       |  720005
+ child1       |  720006
+ child1       |  720007
+ child2       |  720008
+ child2       |  720009
+ child2       |  720010
+ child2       |  720011
+(12 rows)
+
+\set VERBOSITY terse
+DROP TABLE child1, parent, child2;
+NOTICE:  dropped object: drop_partitioned_table.parent original: t normal: f
+NOTICE:  dropped object: drop_partitioned_table.child2 original: t normal: f
+NOTICE:  dropped object: drop_partitioned_table.child1 original: t normal: f
+\d
+      List of relations
+ Schema | Name | Type | Owner
+---------------------------------------------------------------------
+(0 rows)
+
+SELECT logicalrelid, shardid FROM pg_dist_shard WHERE shardid >= 720000 AND shardid < 730000 ORDER BY shardid;
+ logicalrelid | shardid
+---------------------------------------------------------------------
+(0 rows)
+
+\c - - - :worker_1_port
+SET search_path = drop_partitioned_table;
+\d
+      List of relations
+ Schema | Name | Type | Owner
+---------------------------------------------------------------------
+(0 rows)
+
+\c - - - :master_port
+SET search_path = drop_partitioned_table;
+SET citus.next_shard_id TO 720000;
+-- CASE 3
+-- DROP OWNED BY role1; Only parent is owned by role1, children are owned by another owner
+CREATE ROLE role1;
+NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
+SELECT run_command_on_workers('CREATE ROLE role1');
+      run_command_on_workers
+---------------------------------------------------------------------
+ (localhost,57637,t,"CREATE ROLE")
+ (localhost,57638,t,"CREATE ROLE")
+(2 rows)
+
+GRANT ALL ON SCHEMA drop_partitioned_table TO role1;
+SET ROLE role1;
+CREATE TABLE drop_partitioned_table.parent (x text, t timestamptz DEFAULT now()) PARTITION BY RANGE (t);
+RESET ROLE;
+CREATE TABLE child1 (x text, t timestamptz DEFAULT now());
+ALTER TABLE parent ATTACH PARTITION child1 FOR VALUES FROM ('2021-05-31') TO ('2021-06-01');
+CREATE TABLE child2 (x text, t timestamptz DEFAULT now());
+ALTER TABLE parent ATTACH PARTITION child2 FOR VALUES FROM ('2021-06-30') TO ('2021-07-01');
+SELECT create_distributed_table('parent','x');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+\d
+                       List of relations
+         Schema         |  Name  |       Type        |  Owner
+---------------------------------------------------------------------
+ drop_partitioned_table | child1 | table             | postgres
+ drop_partitioned_table | child2 | table             | postgres
+ drop_partitioned_table | parent | partitioned table | role1
+(3 rows)
+
+SELECT logicalrelid, shardid FROM pg_dist_shard WHERE shardid >= 720000 AND shardid < 730000 ORDER BY shardid;
```

Comment:
what is the point of printing pg_dist_shard output on this test?

Reply:
Hmm I believe there is no point. We always delete shard metadata like before.